example/app-router/app/page.tsx

```diff
@@ -0,0 +1,359 @@
+"use client";
+import React, { useState, useRef } from "react";
+import { IKImage, ImageKitProvider, IKUpload, IKVideo } from "../../../src/index";
+import { AbortableFileInput, IKUploadResponse, UploadError } from "../../../src/components/IKUpload/props";
```

Comment:
these should be imported from imagekit-next, not relative paths

Reply:
done